examples/resources/snowflake_stage/resource.tf

```diff
@@ -6,11 +6,3 @@ resource "snowflake_stage" "example_stage" {
   schema      = "EXAMPLE_SCHEMA"
   credentials = "AWS_KEY_ID='${var.example_aws_key_id}' AWS_SECRET_KEY='${var.example_aws_secret_key}'"
 }
-
-resource "snowflake_stage_grant" "grant_example_stage" {
```

Comment:
Should we remove this ownership grant or add todo for SNOW-991423 (grant ownership implementation)?

Reply:
No other resource have such an example, so my choice was to remove and not do a follo-up just for this one